circle.yml

```diff
@@ -0,0 +1,82 @@
+version: 2
+workflows:
+  version: 2
+  linux-builds:
+    jobs:
+      - build-x86_64
+      - build-i686
+
+jobs:
+  build-x86_64:
+    docker:
+#      - image: ubuntu:14.04
+      - image: circleci/python:2.7
+        environment:
+          JULIA_CPU_CORES: 4
```

Comment:
I thought they still used 2 vCPU containers though? Idk, I guess at this point you'd know better than I would.